src/Umbraco.Core/Services/IUserService.cs

```diff
@@ -119,6 +119,13 @@ public interface IUserService : IMembershipUserService
     /// </returns>
     IProfile? GetProfileByUserName(string username);
 
+    /// <summary>
+    /// Get a user by its key.
+    /// </summary>
+    /// <param name="key">The GUID key of the user.</param>
+    /// <returns>The found user, or null if nothing was found.</returns>
+    Task<IUser?> GetAsync(Guid key) => throw new NotImplementedException();
```

Comment:
Same logic as earlier.. A bad implementation is better than throwing. Hopefully nobody have implemented their own UserService.

```suggestion
    Task<IUser?> GetAsync(Guid key) => Task.FromResult(GetAll(0, int.MaxValue, out _).FirstOrDefault(x=>x.Key == key));
```